test/sanitizer_suppressions/ubsan

```diff
@@ -73,3 +74,9 @@ implicit-signed-integer-truncation:test/skiplist_tests.cpp
 implicit-signed-integer-truncation:torcontrol.cpp
 implicit-unsigned-integer-truncation:crypto/*
 implicit-unsigned-integer-truncation:leveldb/*
+shift-base:arith_uint256.cpp
+shift-base:crypto/
+shift-base:hash.cpp
+shift-base:streams.h
+shift-base:util/bip32.cpp
```

Comment:
lines with `arith_uint256...bip32` seems unrelated to backport.

Reply:
@ogabrielides why?